justfile

```diff
@@ -2,11 +2,6 @@
 default:
   just --list
 
-#Rust fmt and linter together
-fl: fmt lint
-setup_cluster: create_k3d_cluster
-
-_cargo := "cargo"
```

Comment:
Why alias this? If you look at the linkerd repo, we may have a `_cargo` var because cargo may not be present. Or, we could have `_cargo-test` because we choose which test harness to run based on what's available in the system. We can reasonably assume cargo is installed here.